addons/l10n_my_ubl_pint/views/report_invoice.xml

```diff
@@ -45,6 +45,39 @@
         </xpath>
     </template>
 
+    <template id="l10n_my_ubl_pint_external_layout_bubble" inherit_id="web.external_layout_bubble">
+        <xpath expr="//ul[@name='company_address_list']" position="inside">
+            <li t-if="company.sst_registration_number and company.account_fiscal_country_id.code == 'MY'">
+                SST: <span t-field="company.sst_registration_number"/>
+            </li>
+            <li t-if="company.ttx_registration_number and company.account_fiscal_country_id.code == 'MY'">
+                TTx: <span t-field="company.ttx_registration_number"/>
+            </li>
+        </xpath>
+    </template>
+
+    <template id="l10n_my_ubl_pint_external_layout_wave" inherit_id="web.external_layout_wave">
+        <xpath expr="//ul[@name='company_address_list']" position="inside">
+            <li t-if="company.sst_registration_number and company.account_fiscal_country_id.code == 'MY'">
+                SST: <span t-field="company.sst_registration_number"/>
+            </li>
+            <li t-if="company.ttx_registration_number and company.account_fiscal_country_id.code == 'MY'">
+                TTx: <span t-field="company.ttx_registration_number"/>
+            </li>
+        </xpath>
+    </template>
+
+    <template id="l10n_my_ubl_pint_external_layout_folder" inherit_id="web.external_layout_folder">
```

Comment:
I don't see an ideal solution because the document height for the header is fixed on the paperformat, a functional (maybe a bit overkilled) implementation to solve this would be to detect the amount of lines on the address block and reduce the font-size accordingly to fit, yet could create the risk of a very small company address. 

In the meantime I think a compromise for this layout is to show the extra info of the layout under its logo, which is not ideal but will at least avoid unreadable content. I don't think it's a big deal, if this layout doesn't fit the user's need they will switch to another document layout or edit the company address in the document_layout configuration.

![image](https://github.com/user-attachments/assets/e2b938f1-5888-4cdb-9022-337513e07465)

Reply:
What you say / your solution makes sense to me (also checked with PO).